packages/flutter/lib/src/rendering/flex.dart

```diff
@@ -597,15 +616,16 @@ class RenderFlex extends RenderBox with ContainerRenderObjectMixin<RenderBox, Fl
 
   double _getIntrinsicSize({
     required Axis sizingDirection,
-    required double extent, // the extent in the direction that isn't the sizing direction
-    required _ChildSizingFunction childSize, // a method to find the size in the sizing direction
+    required double extent, // The extent in the direction that isn't the sizing direction.
+    required double spacing, // The spacing between children.
+    required _ChildSizingFunction childSize, // A method to find the size in the sizing direction.
   }) {
     if (_direction == sizingDirection) {
       // INTRINSIC MAIN SIZE
       // Intrinsic main size is the smallest size the flex container can take
       // while maintaining the min/max-content contributions of its flex items.
       double totalFlex = 0.0;
-      double inflexibleSpace = 0.0;
+      double inflexibleSpace = spacing * (childCount - 1);
```

Comment:
It's weird that you use the `childCount` member from the object but take the `spacing` as a parameter, given that there's also a `spacing` field and that's the only value you ever pass in for spacing. Is there any reason not to just use the `spacing` member directly?

Reply:
Ahh.. excellent catch! I didn't realize I'm passing spacing the field. 
I was just thinking I need to pass spacing and added parameter out of habit.

Reply:
Removed the extra spacing parameter.